astroquery/alma/core.py

```diff
@@ -594,6 +594,11 @@ def query_async(self, payload, *, public=True, science=True,
         if science is not None:
             payload['science_observation'] = science
         if public is not None:
+            if 'public_data' in kwargs:
```

Comment:
This is why I really dislike `**kwargs`, it swallows stuff and creates inconsistencies like this almost duplicate of kwargs with the same functionality.

Reply:
I hear you, but I greatly prefer this over `query(payload={'public_data' : True})`.  I find it really clunky & annoying to specify keywords as dicts.

The redundancy here is almost certainly my fault: I told @andamian that he should preserve as much of the old API as possible, so this is (probably, I'm going from memory here) an API backward-compatibility-driven choice.

Reply:
Redundancy was an agreed tradeoff to make the transition from the Web form to `astroquery` easier. Do you think it's time to remove `public_data` from the help to avoid further confusion?

Reply:
no, I think the redundancy is important here, we just needed the documentation to be clear